articles/active-directory/develop/quickstarts/active-directory-windesktop.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Azure AD v2 Windows desktop quickstart | Microsoft Docs
+description: Learn how a Windows Desktop .NET (XAML) application can get an access token and call an API protected by an Azure Active Directory v2 endpoint
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.assetid: 820acdb7-d316-4c3b-8de9-79df48ba3b06
+ms.service: active-directory
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/15/2018
+ms.author: andret
+ms.custom: aaddev 
+
+---
+
+# Acquire a token and call Microsoft Graph API from a Windows Desktop app
+
+In this quickstart, you'll learn a Windows Desktop .NET (WPF) application can sign in personal, work and school accounts, get an access token, and call the Microsoft Graph API.
+
+![How the sample app generated by this quickstart works](media/active-directory-windesktop/windesktop-intro.png)
+
+
+
+> [!div renderon="docs"]
+> ## Register your application and download your quickstart app
+> You have two options to start your quickstart application:
+> * [Express] [Option 1: Register and auto configure your app and then download your code sample](#option-1-register-and-auto-configure-your-app-and-then-download-your-code-sample)
+> * [Manual] [Option 2: Register and manually configure your application and code sample](#option-2-register-and-manually-configure-your-application-and-code-sample)
+>
+> ### Option 1: Register and auto configure your app and then download your code sample
+>
+> 1. Go to the [Azure portal - Application Registration](https://portal.azure.com/signin/index/?Microsoft_AAD_RegisteredApps=true#blade/Microsoft_AAD_RegisteredApps/applicationsListBlade/quickStartType/WinDesktopQuickstartPage/sourceType/docs).
+> 1. Enter a name for your application and click **Register**.
+> 1. Follow the instructions to download and automatically configure your new application for you in one click.
+>
+> ### Option 2: Register and manually configure your application and code sample
+>
+> #### Step 1: Register your application
+> 
+> 1. To register an application for this code sample, go to the [Azure portal - Application Registration](https://aka.ms/registeredappsprod) and select **New registration**.
+> 1. Enter a name for your application, add `urn:ietf:wg:oauth:2.0:oob` in **Reply URL**, and click **Register**.
+
+> [!div renderon="portal" class="sxs-lookup"]
+> #### Step 1: Configure your application
+> For the code sample for this quickstart to work, you need to add a reply URL as **urn:ietf:wg:oauth:2.0:oob**.
```

Comment:
should we still recommend customers to use `urn:ietf:wg:oauth:2.0:oob` given the upcoming changes in STS and app model?